serial_protocol.md

```diff
@@ -186,3 +192,135 @@ Used to stop listening to events at a given path. This must be the same path pro
 	>>BEGIN_STREAM /user/aturing
 	>>END_STREAM /user/aturing
 	<<+OK
+
+#Messaging Protocol
+
+## BEGIN_MESSAGING 
+Called to start communicating with Firebase Cloud Messaging, it requires a server key.
+###Usage
+	BEGIN %Server_Key%
+###Response
+	+OK - Accepted initialization parameters
+###Examples
+	>> BEGIN https://samplechat.firebaseio-demo.com
+	<< +OK
+
+## MESSAGE_TO_USERS
+Called to start composing a message to users by registration id. You can specify as many as you wish separated by spaces on the same line. 
```

Comment:
We could, I am trying to think through what the protocol would look like then.

I guess I think of the notification topic/target users as closer to the database path for a SET or GET. In truth many IoT applications will write the the same path everytime as well but they still specify it on every command. So what you are describing is closer to your simplified serial protocol for the database, where whatever you write to the line goes to the db.